axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceLineController.java

```diff
@@ -431,34 +428,13 @@ public void setAxisDomains(ActionRequest request, ActionResponse response) {
     try {
       InvoiceLine invoiceLine = request.getContext().asType(InvoiceLine.class);
       if (request.getContext().getParent() != null) {
-        Invoice invoice = request.getContext().getParent().asType(Invoice.class);
-        List<Long> analyticAccountList = new ArrayList<Long>();
-        AnalyticToolService analyticToolService = Beans.get(AnalyticToolService.class);
-        AnalyticLineService analyticLineService = Beans.get(AnalyticLineService.class);
-        for (int i = startAxisPosition; i <= endAxisPosition; i++) {
-          if (invoice != null
-              && invoice.getCompany() != null
-              && analyticToolService.isPositionUnderAnalyticAxisSelect(invoice.getCompany(), i)) {
-            analyticAccountList =
-                analyticLineService.getAxisDomains(invoiceLine, invoice.getCompany(), i);
-            if (CollectionUtils.isEmpty(analyticAccountList)) {
-              response.setAttr(
-                  "axis".concat(Integer.toString(i)).concat("AnalyticAccount"),
-                  "domain",
-                  "self.id IN (0)");
-            } else {
-              String idList =
-                  analyticAccountList.stream()
-                      .map(id -> id.toString())
-                      .collect(Collectors.joining(","));
-              response.setAttr(
-                  "axis".concat(Integer.toString(i)).concat("AnalyticAccount"),
-                  "domain",
-                  "self.id IN (" + idList + ")");
-            }
-          }
-        }
+        return;
```

Comment:
I think it's weird that method is stopped if parent context exists, shouldn't it be the opposite since parent context is used afterwards ?